typing_tests/pine-params.ts

```diff
@@ -211,13 +157,13 @@ await (async () => {
 	});
 	aNumber = result.id;
 	aString = result.device_name;
-	aNumber = result.belongs_to__application.__id;
+	aNumberOrUndefined = result.belongs_to__application?.__id;
 	aNumberOrUndefined = result.is_pinned_on__release?.__id;
 
 	// @ts-expect-error test case
 	aNumber = result.is_pinned_on__release.__id;
 	// @ts-expect-error test case
-	aAny = result.device_tag;
+	anArray = result.device_tag;
```

Comment:
Like in the other file 🙏 
```suggestion
	// TODO: Ideally the prop shouldn't be there at all
	anUndefined = result.device_tag;
```